db/migrate/20220202043722_create_phones.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePhones < ActiveRecord::Migration[6.1]
+  def change
+    create_table :phones do |t|
+      t.string :number
+      t.string :kind
+      t.boolean :main
+      t.references :contact, null: false, foreign_key: true
+
+      t.timestamps
+    end
```

Comment:
It's also a good practice to have a unique index to enforce at the database level the uniqueness validations added at the model level. You could create a compound unique index **after** the table creation with
```rb
add_index :phones, [:kind, :contact_id], unique: true
```